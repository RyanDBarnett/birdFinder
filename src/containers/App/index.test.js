import React from 'react'; 
import { shallow } from 'enzyme';
import { App } from '.';


describe('App', () => {
  let wrapper;
  const fetchSightings = jest.fn(); 
  it('should match the snapshot', () => { 
    wrapper = shallow( <App fetchSightings={fetchSightings} />);

    expect(wrapper).toMatchSnapshot(); 
  });

  it('should have match snapshot if isLoading is true', () => { const
    wrapper = shallow( <App fetchSightings={fetchSightings} isLoading={true} />);

    expect(wrapper).toMatchSnapshot(); 
  });

  it('should have match snapshot if there is an error', () => { const
    wrapper = shallow( <App fetchSightings={fetchSightings} error={'Error'} />);

    expect(wrapper).toMatchSnapshot(); 
  }); 
});