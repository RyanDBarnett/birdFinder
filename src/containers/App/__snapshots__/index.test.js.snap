// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should have match snapshot if isLoading is true 1`] = `ShallowWrapper {}`;

exports[`App should have match snapshot if there is an error 1`] = `ShallowWrapper {}`;

exports[`App should match the snapshot 1`] = `ShallowWrapper {}`;
